Dockerfile.builder

```diff
@@ -1,18 +1,18 @@
-# Used for compiling the tectonic-console. After editing this file,
+# Used for testing OpenShift console in CI. After editing this file,
```

Comment:
`./push-builder.sh` requires a tag, we manually have to bump.  Looks like we just increment by 1 ever time, nothing fancy.  Prob no need to say more in this comment (I'm self documenting a bit while reviewing, since I haven't looked at this file in a while).

Reply:
Thanks, I updated the comment